conferences/ticketing_storage.go

```diff
@@ -0,0 +1,645 @@
+package conferences
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"encore.dev/storage/sqldb"
+	"github.com/lib/pq"
+)
+
+// createAttendee creates a new attendee in the database and returns it.
+func createAttendee(ctx context.Context, tx *sqldb.Tx, a *Attendee) (*Attendee, error) {
+	result := Attendee{}
+	sqlStatement := "INSERT INTO attendee (email, coc_accepted) VALUES ($1, $2) RETURNING id, email, coc_accepted"
+	sqlArgs := []interface{}{a.Email, a.CoCAccepted}
+	var row *sqldb.Row
+
+	if tx != nil {
+		row = sqldb.QueryRowTx(tx, ctx, sqlStatement, sqlArgs...)
+	} else {
+		row = sqldb.QueryRow(ctx, sqlStatement, sqlArgs...)
+	}
+	if err := row.Scan(&result.ID, &result.Email, &result.CoCAccepted); err != nil {
+		return nil, fmt.Errorf("creating or fetching attendee: %w", err)
+	}
+
+	return &result, nil
+}
+
+// readAttendeeByEmail returns an attendee for that email if one exists.
+func readAttendeeByEmail(ctx context.Context, tx *sqldb.Tx, email string) (*Attendee, error) {
+	if email == "" {
+		return nil, fmt.Errorf("email is empty")
+	}
+	return readAttendee(ctx, tx, email, 0)
+}
+
+// readAttendeeByID returns an attendee for the given ID if one exists.
+func readAttendeeByID(ctx context.Context, tx *sqldb.Tx, id int64) (*Attendee, error) {
+	if id == 0 {
+		return nil, fmt.Errorf("id is not valid")
+	}
+	return readAttendee(ctx, tx, "", id)
+}
+
+func readAttendee(ctx context.Context, tx *sqldb.Tx, email string, id int64) (*Attendee, error) {
+	results := Attendee{}
+	sqlStatement := `SELECT id, email, coc_accepted FROM attendee`
+	sqlArgs := []interface{}{}
+	if email != "" {
+		sqlStatement = `SELECT id, email, coc_accepted FROM attendee WHERE email = $1`
+		sqlArgs = append(sqlArgs, email)
+	}
+	if email != "" && id != 0 {
+		sqlStatement = `SELECT id, email, coc_accepted FROM attendee WHERE email = $1 AND id = $2`
+		sqlArgs = append(sqlArgs, id)
```

Comment:
This should have both email and ID I think?
```suggestion
		sqlArgs = append(sqlArgs, email, id)
```